lib/SimpleSAML/XHTML/Template.php

```diff
@@ -162,12 +162,12 @@ public function asset(string $asset, string $module = null): string
     {
         $baseDir = $this->configuration->getBaseDir();
         if (is_null($module)) {
-            $file = $baseDir . 'www/assets/' . $asset;
+            $file = $baseDir . 'www/' . $asset;
```

Comment:
I use `symfony/asset` module that implements `asset` function in twig. This modification is to have the same behaviour in the old a new way to read templates.